lib/ansible/modules/database/postgresql/postgresql_table.py

```diff
@@ -0,0 +1,661 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: postgresql_table
+short_description: Allow to create, drop, rename, or truncate a table and to change some table attributes
```

Comment:
Needs `Postgresql` to appear in here somewhere, and maybe shortening a bit
Maybe change to `Create or modify Postgresql table attributes`

Reply:
Yes, I agree, it looks quite complex now